.github/workflows/step_test.yaml

```diff
@@ -32,12 +30,15 @@ jobs:
       fail-fast: false
       matrix:
         toolchain: [ gcc, llvm, intel, windows, macos ]
-        python-version: [ "3.9", "3.10", "3.11"]
+        python-version: [ "3.8", "3.9", "3.10", "3.11", "3.12"]
```

Comment:
It is in `experimental` at the moment, and it will probably be fixed in a patch version that will be picked up automatically. Let's keep it in for now and have upstream check up on it as well if needed.